cmd/drone-server/drone.go

```diff
@@ -185,6 +185,14 @@ func main() {
 			repo.GET("/logs/:number/:task", server.GetLogs)
 			// repo.POST("/status/:number", server.PostBuildStatus)
 		}
+
+		// Routes for external services
+		repoExternal := repos.Group("")
+		{
+			repoExternal.Use(server.SetRepo())
+
+			repoExternal.GET("/pr/:number", server.GetPullRequest)
```

Comment:
let's do `/pulls/:number` to more closely align with GItHub, if you don't mind :)

also, this can be put in the above group. The reason is that none of the middleware (security, etc) will be applied unless it sits in its own security group.

```diff
			repo.GET("/logs/:number/:task", server.GetLogs)
-		}
-
-		// Routes for external services
-		repoExternal := repos.Group("")
-		{
-			repoExternal.Use(server.SetRepo())
-			repoExternal.GET("/pr/:number", server.GetPullRequest)
+			repo.GET("/pulls/:number", server.GetPullRequest)
+		}
```

we'll also be able to add `repoExternal.GET("/pulls", server.GetPullRequests)` which I know @donny-dont would like as well